app/controllers/recipes_controller.rb

```diff
@@ -9,10 +9,14 @@ def new
     @recipe = Recipe.new
   end
 
+  def show
+    @recipe = Recipe.find(params[:id])
```

Comment:
This is okey, but maybe we could use  `@recipe = current_user.recipes.find(params[:id])` that way we ensure that we only show it if it is the current user's recipe, what do you think?